dbms/src/Common/MemoryTracker.cpp

```diff
@@ -228,6 +256,22 @@ thread_local MemoryTracker * current_memory_tracker = nullptr;
 std::shared_ptr<MemoryTracker> root_of_non_query_mem_trackers = MemoryTracker::createGlobalRoot();
 std::shared_ptr<MemoryTracker> root_of_query_mem_trackers = MemoryTracker::createGlobalRoot();
 
+std::shared_ptr<MemoryTracker> sub_root_of_query_storage_task_mem_trackers;
+std::shared_ptr<MemoryTracker> rn_fetch_pages_mem_tracker;
+
+void initMemoryTracker()
+{
+    RUNTIME_CHECK(sub_root_of_query_storage_task_mem_trackers == nullptr);
+    sub_root_of_query_storage_task_mem_trackers = MemoryTracker::create();
+    sub_root_of_query_storage_task_mem_trackers->setNext(root_of_query_mem_trackers.get());
```

Comment:
`root_of_query_mem_trackers` will be reset when there is no active queries, I'm not sure if it still ok to be reset after this pr?

Reply:
when memtrackers of query_storage and page are reset，are their memory really released？